rpi_backlight.py

```diff
@@ -155,10 +174,13 @@ def cli() -> None:
         for arg in (
             args.off,
             args.on,
+            args.toggle,
             args.brightness,
             args.max_brightness,
             args.actual_brightness,
             args.power,
+            args.duration, 
+            args.smooth,
```

Comment:
These two lines don't make sense! It's perfectly ok to have smooth `False`, and duration defaults to 0 and doen't belong here as well.

Reply:
Okay I remember why I did this. This was so something like this would work:
rpi-backlight -b 0
Without these 2 lines, this command would show the help menu. I've re-added these 2 lines.